installing/installing_with_agent_based_installer/installing-with-agent-based-installer.adoc

```diff
@@ -18,6 +18,7 @@ include::modules/installing-ocp-agent.adoc[leveloffset=+1]
 .Additional resources
 
 * See xref:../../installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc#installation-three-node-cluster_installing-restricted-networks-bare-metal[Configuring a three-node cluster] to deploy three-node clusters in bare metal environments.
+* See xref:../../installing/installing_bare_metal_ipi/ipi-install-installation-workflow.adoc#root-device-hints_ipi-install-installation-workflow[Root device hints].  
```

Comment:
It might actually make sense to copy-and-paste this table. There are two fields supported in IPI (`wwnVendorExtension` and `wwnWithExtension`) that are _not_ supported in the agent installer. They (mostly) share an API, but not an implementation, so if more were added to IPI we also wouldn't pick them up automatically.